src/utils.hpp

```diff
@@ -460,7 +460,7 @@ extern bool PLR_TOG_CHK(char_data *ch, dword offset);
 #define GET_ESSHOLE(ch)       ((ch)->real_abils.esshole)
 #define GET_INDEX(ch)         ((ch)->real_abils.bod_index)
 #define GET_HIGHEST_INDEX(ch)      ((ch)->real_abils.highestindex)
-#define GET_BIOOVER(ch)       (-((int)((GET_ESS((ch)) + 300) - GET_INDEX((ch))) / 100))
```

Comment:
What's the goal of adding 99 here? Ensuring that we round up instead of down? If so, does this not compound with the other changes you made that also added +1 to the bio-over value?

Reply:
Right, the +99 here is to round up. The combination with other places is needed to match the rules as written.

Drug(toxin) resistance rule: "lose 1 die on Resistance Tests for every 2 points (or part thereof) their Bio Index exceeds their Essence Index" (M&M pg78). Checking the rounding limits:

1. Exceeding by 0.01 makes this part (1+99)/100 = 1, then the drug resist part is (1+1)/2 = 1.
2. Exceeding by 2.00 makes this part (200+99)/100 = 2, then the drug resist part is (2+1)/2 = 1.

Because of integer truncation, both parts need to be rounded up to make the 0.01 case cause a penalty, and rounding up both parts won't cause the 2.00 case to give too high of a penalty.